wiki/Welcome/en.md

```diff
@@ -1,148 +1,37 @@
-
-### Performance Points
-*Full explanation: [Performance Points][Performance_Points wikilink]*
-
-Performance Points (pp) is a ranking metric aiming to be more contextually relevant to a continuous game like osu!. It aims to shift the focus from being on the amount of time played to the actual skill of the player. Performance points are heavily based on map difficulty which is determined by a unique algorithm in every gamemode. Based on this difficulty alone each of your scores is rated and assigned a pp value. Your performance rank is predominantly based on your performance on individual maps. The easiest way to improve it is to improve your scores on difficult songs. Do note that game modes are counted separately \[which means, playing (osu!/osu!taiko/osu!catch/osu!mania) raises (osu!/osu!taiko/osu!catch/osu!mania) pp\].
-
-If you have any questions regarding pp, please refer to the [wiki page for pp first][Performance_Points#FAQ wikilink].
-
-### Accuracy
-_Full explanation: [Accuracy][Accuracy wikilink]_
 
-![A screenshot depicting a player&#39;s accuracy after completing a beatmap.][Accuracy results image]
+Welcome to _osu!_, a free-to-win rhythm game developed by peppy with four game modes: osu!standard, a circle clicking simulator; osu!taiko, a drumming emulator; osu!catch, a fruit salad catcher; osu!mania, a key smashing mania.
 
-This measurement is the average accuracy among all the ranked scores the player played. It is split into two types, player's overall and pp's accuracy. In your player tab in-game, the accuracy shown is pp's accuracy. For player's overall, you may have to visit the "Online Results" after you played a beatmap online or refer to your own forum page. In osu!, accuracy measures not only your aim, but the timing as well, so it is important to tap the notes perfectly to the beat. Pay extra attention to the approach circles!
+## Current events
 
-### Achievements
-_Full explanation: [Achievements][Achievements wikilink]_
+<!--There are no active events within the osu! community at this time.-->
+- [Aspire Stage Three](https://osu.ppy.sh/home/news/2017-09-21-aspire-stage-three-osucatch-begins)
+- [osu!idol 2017 (Auditions)](https://osu.ppy.sh/home/news/2017-08-14-osu-idol-2017-auditions-now-open)
+- [Yuzu's New Look Contest](https://osu.ppy.sh/home/news/2017-08-21-yuzus-new-look-contest)
 
-Achievements are little badges on one's profile which indicate that someone has accomplished an exceptional goal. Achievements' badge can be unlocked in various ways such as playing a lot or hitting a certain combo count in one beatmap. When you fulfilled the criteria, you will receive a badge at "Online Results" and your account's personal achievement tab. However, the meat of it is the hidden mission, Hush-Hush, where the criteria is not known and it is up to us to figure it out on how to get it. They are pretty hard to get but it is fulfilling when you managed to achieved it by yourself.
+## Ranking process
```

Comment:
why does this header even exists **here**

Reply:
Also, this section is incomplete, but that's because I have never completed the beatmap ranking procedure (tho, I assume it has something to do with BNs and QATs). So someone else that knows it well should review comment or fix it later.